qemu/qemu.go

```diff
@@ -2568,6 +2615,14 @@ func (config *Config) appendLogFile() {
 	}
 }
 
+func (config *Config) appendFwCfg() {
+	if !config.FwCfg.Valid() {
+		return
```

Comment:
please print an error before returning (QMPLog)

Reply:
error printed. not sure about the text. Please advice here